pkg/apiserver/info/info.go

```diff
@@ -110,6 +94,37 @@ func (s *Service) infoHandler(c *gin.Context) {
 	c.JSON(http.StatusOK, resp)
 }
 
+func (s *Service) checkNgmState() (utils.NgmState, error) {
+	ngmState := utils.NgmStateNotSupported
+
+	// The FeatureVersion value is "N/A" when running `make && bin/tidb-dashboard`
+	// The FeatureVersion value is "Unknown" when running `go run cmd/tidb-dashboard/main.go`
+	featureSupported :=
+		s.params.Config.FeatureVersion == "N/A" ||
+			s.params.Config.FeatureVersion == "Unknown"
+	if !featureSupported {
+		// drop "-alpha-xxx" suffix
+		versionWithoutSuffix := strings.Split(s.params.Config.FeatureVersion, "-")[0]
+		v, err := semver.NewVersion(versionWithoutSuffix)
+		if err != nil {
+			return ngmState, err
+		}
+		constraint, _ := semver.NewConstraint(">= v5.4.0")
```

Comment:
Why do we need to hard-code a 5.4.0 constraint here?

Reply:
@shhdgit why we don't register it to the feature flag?

Reply:
Yes, we can use the feature flag to declare whether ngm is supported or not.